### PR TITLE
Keybase fixes

### PR DESCRIPTION
**Pull request by mrsmkl**
### Description

Looks like more fixes are needed to get keybase working.

### CODE REVIEW

**Review by nambrot — APPROVED** [review]

File: packages/cli/src/utils/identity.ts [hunk 1]
```diff
@@ -34,6 +35,7 @@ export abstract class ClaimCommand extends BaseCommand {
     try {
       cli.action.start(`Read Metadata from ${filePath}`)
       const data = IdentityMetadataWrapper.fromFile(this.kit, filePath)
+      // Perhaps should check here that `--from` and data.data.meta.address match
```
Comment (nambrot) [inline packages/cli/src/utils/identity.ts]: That's no longer the case right? From can be a signer?

Comment (mrsmkl) [reply packages/cli/src/utils/identity.ts]: Yeah, so should check that it's the address or one of the signers.

File: packages/cli/src/utils/identity.ts [hunk 2]
```diff
@@ -20,20 +21,37 @@ export abstract class ClaimCommand extends BaseCommand {
...
+  }
+
+  protected readMetadata = async () => {
+    const { args, flags } = this.parse(this.self)
     const filePath = args.file
     try {
       cli.action.start(`Read Metadata from ${filePath}`)
-      const data = IdentityMetadataWrapper.fromFile(this.kit, filePath)
+      const data = await IdentityMetadataWrapper.fromFile(this.kit, filePath)
+      if (flags.from) {
```
Comment (nambrot) [inline packages/cli/src/utils/identity.ts]: This is required right, so you can just assume it is there?

File: packages/contractkit/src/identity/metadata.ts [hunk 3]
```diff
@@ -139,11 +139,19 @@ export class IdentityMetadataWrapper {
...
         break
-      case ClaimTypes.DOMAIN:
+      case ClaimTypes.DOMAIN: {
         const existingClaims = this.data.claims.filter((el: any) => el.domain === claim.domain)
         if (existingClaims.length > 0) {
           return existingClaims[0]
         }
+        break
+      }
+      case ClaimTypes.KEYBASE: {
```
Comment (nambrot) [inline packages/contractkit/src/identity/metadata.ts:154]: One should be able to claim multiple keybase accounts, right?

Comment (mrsmkl) [reply packages/contractkit/src/identity/metadata.ts:154]: The idea here is to have only claim for each username, similarly to the domains above.

Comment (nambrot) [reply packages/contractkit/src/identity/metadata.ts:154]: Ah right my bad

**Review by nambrot — APPROVED** [review]

**Pull request merged**